server/src/test/java/org/elasticsearch/search/internal/ShardSearchRequestTests.java

```diff
@@ -257,11 +258,11 @@ public void testWillCallRequestCacheKeyDifferentiators() throws IOException {
         assertThat(invoked.get(), is(true));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/96910")
     public void testForceSyntheticUnsupported() throws IOException {
         SearchRequest request = createSearchRequest();
         if (request.source() != null) {
             request.source().rankBuilder(null);
+            request.source().subSearches(new ArrayList<>());
```

Comment:
We shouldn't do this. It will remove all coverage for when there is a single query. Does this test fail when there is just one subSearch (e.g. just a query?)

Reply:
Good point! I'll only remove it if there size >= 2.